indico/modules/events/editing/client/js/components/EditableSubmissionButton.jsx

```diff
@@ -18,14 +18,18 @@ import {Translate} from 'indico/react/i18n';
 import {FinalFileManager} from './FileManager';
 import {fileTypePropTypes} from './FileManager/util';
 
+import * as selectors from '../selectors';
```

Comment:
I'd just import the function itself here since we aren't using many different selectors